tasks/apply-mapping/apply-mapping.yaml

```diff
@@ -86,6 +89,13 @@ spec:
                 variable=$(jq -c --argjson i "$i" '.[$i]' <<< "${SUPPORTED_VARIABLES}")
                 KEY=$(jq -r 'to_entries[] | .key' <<< "$variable")
                 VALUE=$(jq -r 'to_entries[] | .value' <<< "$variable")
+                # If a tag is going to be replaced by a null value, error out
+                if [[ "$VALUE" == "null" ]] && \
+                  [[ $(jq --arg key "$KEY" '. | map(select(test($key))) | length' <<< "$tags") -gt 0 ]] ; then
+                    echo "Requested substitution for key: $KEY but calculated value for it was: $VALUE"
```

Comment:
Optional: This check is specifically for "null", so I think just printing "null" directly would imo be more straightforward. But if you prefer this, it's fine.

Reply:
I updated it